persistence/pre_persistence/proto/account.proto

```diff
@@ -1,13 +1,16 @@
 syntax = "proto3";
-package utility;
-option go_package = "/utility/test/";
+package pre_persistence;
+
+option go_package = "/persistence/pre_persistence/";
 
 message Account {
-  bytes Address = 1;
-  string Amount = 2;
+  bytes address = 1;
+  string amount = 2;
```

Comment:
```suggestion
  string balance = 2;
```

Wdyt?

Reply:
I prefer amount, but this is a non-trivial naming change that we should probably discuss if it's a contention

Reply:
The `balance` of an `account` is more semantic in my opinion. Feel free to briefly highlight why it's a non-trivial change and why `amount` would make more sense. Not a point of contention per se but worth a quick visit.

Reply:
Noted